junebug/channel.py

```diff
@@ -142,3 +152,42 @@ def status(self):
         # TODO: Implement channel status
         status['status'] = {}
         return status
+
+    def _api_from_message(self, msg):
+        ret = {}
+        ret['to'] = msg['to_addr']
+        ret['from'] = msg['from_addr']
+        ret['message_id'] = msg['message_id']
+        ret['channel_id'] = msg['transport_name']
+        ret['timestamp'] = msg['timestamp']
+        ret['reply_to'] = msg['in_reply_to']
+        ret['content'] = msg['content']
+        ret['session_event'] = msg['session_event']
+        ret['channel_data'] = msg['helper_metadata']
+        if not ret['channel_data']:
+            ret['channel_data'] = {}
+        if msg.get('continue_session'):
+            ret['channel_data']['continue_session'] = msg['continue_session']
+        return ret
+
+    def _message_from_api(self, msg):
+        ret = {}
+        ret['to_addr'] = msg.get('to')
+        ret['from_addr'] = msg['from']
+        ret['content'] = msg['content']
+        ret['transport_name'] = self.id
+        channel_data = msg.get('channel_data', {})
+        if channel_data.get('continue_session'):
+            ret['continue_session'] = channel_data.pop('continue_session')
```

Comment:
Similar comment to [here](https://github.com/praekelt/junebug/pull/9/files#r38942256).